src/java.compiler/share/classes/javax/lang/model/AnnotatedConstruct.java

```diff
@@ -189,10 +189,11 @@ public interface AnnotatedConstruct {
     <A extends Annotation> A getAnnotation(Class<A> annotationType);
 
     /**
-     * Returns annotations that are <em>associated</em> with this construct.
+     * Returns annotations of the specified type that are <em>associated</em>
```

Comment:
```suggestion
     * Returns annotations of the specified interface that are <em>associated</em>
```

Reply:
Please see the entire javadoc, not just the parts ended up in this diff. There's one more method that uses this term. In general, "type" should not always be changed to "interface" in the context of annotations.